src/po/Makefile

```diff
@@ -231,7 +262,10 @@ gvim.desktop: gvim.desktop.in $(POFILES) vim.desktop
 	if command -v desktop-file-validate; then desktop-file-validate tmp_gvim.desktop; fi
 	mv tmp_gvim.desktop gvim.desktop
 
-update-po: $(LANGUAGES)
+# 06.11.23, changed by Restorer
+# When updating ja.sjis.po there are a bunch of errors and a crash.
+# The files that are converted to a different encoding clearly state "DO NOT EDIT".
+update-po : $(MOFILES:.mo=)
```

Comment:
Changing `$(LANGUAGES)` to `$(MOFILES:.mo=)` doesn't look correct.
Could you explain this?

Reply:
Okay, understood.
`update-po` is used to update the original translation files based on the latest source code.
Converted po files are updated by other rules.

Then, the following part
```
# Don't add a dependency here, we only want to update the .po files manually
$(LANGUAGES):
```
should be also updated like this?
```
# Don't add a dependency here, we only want to update the .po files manually
$(MOFILES:.mo=):
```

Reply:
Yes, this is how I originally wanted to do it, but unfortunately it doesn't work quite right. For example, when the command to update a single language from the command line is executed.
`nmake -f Make_mvc.mak <LNG>`
And secondly, we still have the option to manually update PO files that are created during conversion. If we need it for some reason.